### ISSUE TITLE
Divide the dataset

### ISSUE THREAD

**Issue by Liu-art6** [dlcjfgmlnasa/SynthSleepNet#3]
The SHHS1 dataset contains data from 100 patients. The dataset was divided into three subsets: training, pre-training, and testing.
According to the code:1.parser.add_argument('--test_size', default=0.10, type=float),testing group include 10 patients.
                                     2.parser.add_argument('--holdout_subject_size', default=50, type=int),training group include 50 patients.
                                     3.so the pre-training group include 50 patients.
But the paper mentioned " The training group, consisting of a small subset of labeled data, was used for linear evaluation and fine-tuning."I'm very confused about this.So I would like to ask you about the setting of these two parameters(test_size and holdout_subject_size).If I understand it wrong, could you explain it to me?Thanks a lot for your help.

**Comment by dlcjfgmlnasa** [comment]
Hello, thank you for your interest in *SynthSleepNet*.

I don't remember exactly, but as far as I know, the SHHS1 dataset includes data from over 5,000 participants.

---

The *holdout_subject* size can be considered as the test set.

If you have any questions, please feel free to ask. Thank you! 👍
